plugins/node/opentelemetry-instrumentation-ioredis/README.md

```diff
@@ -85,6 +114,6 @@ Apache 2.0 - See [LICENSE][license-url] for more information.
 [dependencies-image]: https://status.david-dm.org/gh/open-telemetry/opentelemetry-js-contrib.svg?path=plugins%2Fnode%2Fopentelemetry-instrumentation-ioredis
 [dependencies-url]: https://david-dm.org/open-telemetry/opentelemetry-js-contrib?path=plugins%2Fnode%2Fopentelemetry-instrumentation-ioredis
 [devdependencies-image]: https://status.david-dm.org/gh/open-telemetry/opentelemetry-js-contrib.svg?path=plugins%2Fnode%2Fopentelemetry-instrumentation-ioredis&type=dev
-[devDependencies-url]: https://david-dm.org/open-telemetry/opentelemetry-js-contrib?path=plugins%2Fnode%2Fopentelemetry-instrumentation-ioredis&type=dev
+[devdependencies-url]: https://david-dm.org/open-telemetry/opentelemetry-js-contrib?path=plugins%2Fnode%2Fopentelemetry-instrumentation-ioredis&type=dev
```

Comment:
hmm why this change ?

Reply:
weird, I don't remember changing it. maybe some linter or something
Anyhow, the current README in main looks broken, at least currently in my browser, so maybe we can keep it? I don't care to revert it also if you prefer.

![image](https://user-images.githubusercontent.com/22870745/116131910-9d2aba00-a6d5-11eb-9f8a-ddeb2dca010a.png)

Reply:
I think there some kind of bug with david dm here: https://david-dm.org/open-telemetry/opentelemetry-js-contrib?path=plugins%2Fnode%2Fopentelemetry-instrumentation-ioredis&type=dev

Reply:
> I think there some kind of bug with david dm here: https://david-dm.org/open-telemetry/opentelemetry-js-contrib?path=plugins%2Fnode%2Fopentelemetry-instrumentation-ioredis&type=dev

I'm not very familiar with this README badges and how they work. @vmarchaud do you know how to fix it? should I revert the README badge change in this PR?

Reply:
No worries, i don't think the problem comes from us anyway so i think you should just revert that changes (unless thats the linter but i don't remember that our linter also touch markdown files)

Reply:
I don't mind reverting it, but the current version in **main** is not consistent. at the top of the README it says:
```
[![devDependencies][devDependencies-image]][devDependencies-url]
```

while at the bottom the image tag is different:
```
[devdependencies-image]: https://status.david-dm.org/gh/open-telemetry/opentelemetry-js-contrib.svg?path=plugins%2Fnode%2Fopentelemetry-instrumentation-ioredis&type=dev
[devDependencies-url]: https://david-dm.org/open-telemetry/opentelemetry-js-contrib?path=plugins%2Fnode%2Fopentelemetry-instrumentation-ioredis&type=dev
```

devdependencies-image is capitalized at one place and lower case at the other...

Reply:
this is a link showing all dependencies but for some reason recently "david" just showing errors randomly. Then it works again etc. I was fixing all the links some time ago and they were working again, now might be either temporary or something else has changed.

Reply:
reverted this change. thanks

Reply:
figured out where it came from.
When I press `option` + `shift` + `F` in visual studio code (which I do regularly) it does this modification to the markdown. Wonder why.
Anyway, it's not relevant to this PR :)